hist/histv7/inc/ROOT/RHistBufferedFill.hxx

```diff
@@ -19,6 +19,9 @@
 #include "ROOT/RSpan.hxx"
 #include <array>
 
+#include <cstddef>
+#include <array>
```

Comment:
```suggestion
```
Array is already included a few lines above. Also, why the line break between <cstddef> and the <array> before? There is also no line break between the other includes

Reply:
Sorry, I was testing this on `6.28.2` and didn't recheck on `master`. I'll fix this.